frontend/src/pages/Dashboard.jsx

```diff
@@ -111,8 +91,8 @@ class Dashboard extends Component {
                 <title>Dashboard | Spoti-Vote</title>
                 <meta name="author" content="Lukas Samir Gabsi, Michael Blank"></meta>
             </Helmet>
-            <Main topTracks={this.state.topTracks} profile={this.state.profile}/>
-            <SharedSidebar profile={this.state.profile}/>
+            <Main topTracks={this.state.topTracks} host={this.state.host}/>
```

Comment:
I see that host is being passed around a lot, maybe we put the host into a context provider? If you think so too please mention it in issue #234 as a reminder for me